src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -12,9 +12,9 @@ azure-core==1.35.0
 azure-cosmos==3.2.0
 azure-data-tables==12.4.0
 azure-datalake-store==1.0.1
-azure-keyvault-administration==4.4.0b2
+azure-keyvault-administration==4.4.0
 azure-keyvault-certificates==4.7.0
-azure-keyvault-keys==4.11.0b1
+azure-keyvault-keys==4.11.0
 azure-keyvault-secrets==4.7.0
 azure-keyvault-securitydomain==1.0.0b1
```

Comment:
`azure-keyvault-securitydomain` doesn't have any stable version so we can't upgrade at the moment